sass/_color.scss

```diff
@@ -0,0 +1,24 @@
+//Define colors here
+
+$white: #FFFFFF;
+
+$primary-font-color: $white;
+
+//night blue background color
+$background-A: #162327;
```

Comment:
Generally speaking, you want to avoid using iterators like `A`, `B`, and `C` in favor of more descriptive naming (`background-hero`, `background-feature`, etc).